doc/lua_api.txt

```diff
@@ -6526,7 +6526,9 @@ Used by `minetest.register_node`.
         drop = "",
         -- Name of dropped item when dug.
         -- Default dropped item is the node itself.
-        -- Using a table allows multiple items, drop chances and tool filtering:
+        -- Using a table allows multiple items, drop chances and tool filtering.
+        -- Tool filtering was undocumented until recently, tool filtering by string
+        -- matching is now deprecated.
```

Comment:
`deprecated since 5.1.0` to be precise. In 5.3.0 modders will still read it as "now", whereas it's already deprecated for a longer time.

Reply:
I'll remove the 'now' word.

Reply:
Done.